src/workloads/scale/LoadTest.yml

```diff
@@ -0,0 +1,119 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-perf"
+Description: |
+  Based on LongLivedTransactions Insert workload. Using to experiment with bulk load for PERF-2330
+
+GlobalDefaults:
+  dbname: &dbname perf2330
+  MaxPhases: &MaxPhases 20
+
+  # The Sample Document Shape.
+  Document: &Doc
+    ts: {^Now: {}}
+    caid: {^RandomInt: {min: 0, max: 1000}}
+    cuid: {^RandomInt: {min: 0, max: 100000}}
+    prod: {^RandomInt: {min: 0, max: 10000}}
+    price: {^RandomDouble: {min: 0.0, max: 1000.0}}
+    data: {^Join: {array: ["aaaaaaaaaa", {^FastRandomString: {length: {^RandomInt: {min: 0, max: 10}}}}]}}
```

Comment:
Why is one a FastRandomString and the next one a RandomString? Seems we should match them up.

Reply:
I changed both to FastRandomString